From 7345dfea41ddf9cafc2b91b5c90f12ca9ceaffd6 Mon Sep 17 00:00:00 2001
From: Matej Tyc <matyc@redhat.com>
Date: Thu, 10 Feb 2022 19:11:57 +0100
Subject: [PATCH] RHEL9 is supported

State that rhel9 will be supported by the vendor (as soon as it starts
to exist)
---
 .../installed_OS_is_vendor_supported/oval/shared.xml             | 1 +
 1 file changed, 1 insertion(+)

diff --git a/linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_vendor_supported/oval/shared.xml b/linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_vendor_supported/oval/shared.xml
index 931be7e8959..16c3847adb7 100644
--- a/linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_vendor_supported/oval/shared.xml
+++ b/linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_vendor_supported/oval/shared.xml
@@ -6,6 +6,7 @@
     <criteria comment="Installed operating system is supported by a vendor" operator="OR">
       <extend_definition comment="Installed OS is RHEL7" definition_ref="installed_OS_is_rhel7" />
       <extend_definition comment="Installed OS is RHEL8" definition_ref="installed_OS_is_rhel8" />
+      <extend_definition comment="Installed OS is RHEL9" definition_ref="installed_OS_is_rhel9" />
       <extend_definition comment="Installed OS is OL7" definition_ref="installed_OS_is_ol7_family" />
       <extend_definition comment="Installed OS is OL8" definition_ref="installed_OS_is_ol8_family" />
       <extend_definition comment="Installed OS is SLE12" definition_ref="installed_OS_is_sle12" />
